torch/testing/_internal/distributed/rpc/rpc_test.py

```diff
@@ -4096,6 +4460,71 @@ def rref_error():
 
         dist.barrier()
 
+    class VanillaPs:
```

Comment:
For consistency with other test classes and method, let's just call it `MyParameterServer`

Reply:
Okay.